### ISSUE TITLE
Raises SyntaxError when parsing files with umlauts.

### ISSUE THREAD

**Issue by tobiasraabe** [m-burst/flake8-pytest-style#54]
# Bug report

## What's wrong

Hi, thanks for the plugin!

I use `flake8` with `flake8-pytest-style` as a pre-commit hook. When I check Python files containing umlauts a `SyntaxError` is raised.

```
  File "C:\Users\tobia\.cache\pre-commit\repo__2jv1f_\py_env-default\Scripts\flake8.EXE\__main__.py", line 9, in <module>
  File "c:\users\tobia\.cache\pre-commit\repo__2jv1f_\py_env-default\lib\site-packages\flake8\main\cli.py", line 22, in main
    app.run(argv)
  File "c:\users\tobia\.cache\pre-commit\repo__2jv1f_\py_env-default\lib\site-packages\flake8\main\application.py", line 360, in run
    self._run(argv)
  File "c:\users\tobia\.cache\pre-commit\repo__2jv1f_\py_env-default\lib\site-packages\flake8\main\application.py", line 348, in _run
    self.run_checks()
  File "c:\users\tobia\.cache\pre-commit\repo__2jv1f_\py_env-default\lib\site-packages\flake8\main\application.py", line 262, in run_checks
    self.file_checker_manager.run()
  File "c:\users\tobia\.cache\pre-commit\repo__2jv1f_\py_env-default\lib\site-packages\flake8\checker.py", line 325, in run
    self.run_serial()
  File "c:\users\tobia\.cache\pre-commit\repo__2jv1f_\py_env-default\lib\site-packages\flake8\checker.py", line 309, in run_serial
    checker.run_checks()
  File "c:\users\tobia\.cache\pre-commit\repo__2jv1f_\py_env-default\lib\site-packages\flake8\checker.py", line 589, in run_checks
    self.run_ast_checks()
  File "c:\users\tobia\.cache\pre-commit\repo__2jv1f_\py_env-default\lib\site-packages\flake8\checker.py", line 496, in run_ast_checks
    for (line_number, offset, text, _) in runner:
  File "c:\users\tobia\.cache\pre-commit\repo__2jv1f_\py_env-default\lib\site-packages\flake8_plugin_utils\plugin.py", line 75, in run
    self._load_file()
  File "c:\users\tobia\.cache\pre-commit\repo__2jv1f_\py_env-default\lib\site-packages\flake8_plugin_utils\plugin.py", line 91, in _load_file
    self._tree = ast.parse(''.join(self._lines))
  File "C:\tools\miniconda3\envs\gettsim\Lib\ast.py", line 47, in parse
    return compile(source, filename, mode, flags,
  File "<unknown>", line 184
    einführungsfaktor = 0.6 + 0.02 * (
                     ^
SyntaxError: invalid character in identifier
```

## How it should work

Parse files correctly.

## System information

* Operating system: Windows
* Python version: 3.8.5
* flake8 version: 3.8.3
* flake8-pytest-style version: 1.2.2

**Comment by m-burst** [comment]
Hi @tobiasraabe,
Thanks for your report. I'll take a closer look at this next week.

**Comment by m-burst** [comment]
Hi @tobiasraabe,
Unfortunately I cannot reproduce the error either on Windows or on Linux.
Could you please create a minimal example reproducing the bug and push it to Github?

**Comment by tobiasraabe** [comment]
Here is the example.

```python
# Content of some utf-8 encoded file.

def func_with_umlauts_äöü():
    pass

```

```yaml
-   repo: https://gitlab.com/pycqa/flake8
    rev: 3.8.3
    hooks:
    -   id: flake8
        additional_dependencies: [
            flake8-pytest-style,  # produces SyntaxError with umlauts.
        ]

```
Then, init repo, add both files, run `pre-commit run -a` and I this error on Windows, but not on Linux.

```
(test) C:\Users\tobia\git\pytest-flake8 [master +2 ~0 -0 ~]> pre-commit run -a
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

Traceback (most recent call last):
  File "C:\tools\miniconda3\envs\test\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\tools\miniconda3\envs\test\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\tobia\.cache\pre-commit\repo3_aw60i6\py_env-default\Scripts\flake8.EXE\__main__.py", line 7, in <module>
  File "c:\users\tobia\.cache\pre-commit\repo3_aw60i6\py_env-default\lib\site-packages\flake8\main\cli.py", line 22, in main
    app.run(argv)
  File "c:\users\tobia\.cache\pre-commit\repo3_aw60i6\py_env-default\lib\site-packages\flake8\main\application.py", line 360, in run
    self._run(argv)
  File "c:\users\tobia\.cache\pre-commit\repo3_aw60i6\py_env-default\lib\site-packages\flake8\main\application.py", line 348, in _run
    self.run_checks()
  File "c:\users\tobia\.cache\pre-commit\repo3_aw60i6\py_env-default\lib\site-packages\flake8\main\application.py", line 262, in run_checks
    self.file_checker_manager.run()
  File "c:\users\tobia\.cache\pre-commit\repo3_aw60i6\py_env-default\lib\site-packages\flake8\checker.py", line 325, in run
    self.run_serial()
  File "c:\users\tobia\.cache\pre-commit\repo3_aw60i6\py_env-default\lib\site-packages\flake8\checker.py", line 309, in run_serial
    checker.run_checks()
  File "c:\users\tobia\.cache\pre-commit\repo3_aw60i6\py_env-default\lib\site-packages\flake8\checker.py", line 589, in run_checks
    self.run_ast_checks()
  File "c:\users\tobia\.cache\pre-commit\repo3_aw60i6\py_env-default\lib\site-packages\flake8\checker.py", line 496, in run_ast_checks
    for (line_number, offset, text, _) in runner:
  File "c:\users\tobia\.cache\pre-commit\repo3_aw60i6\py_env-default\lib\site-packages\flake8_plugin_utils\plugin.py", line 75, in run
    self._load_file()
  File "c:\users\tobia\.cache\pre-commit\repo3_aw60i6\py_env-default\lib\site-packages\flake8_plugin_utils\plugin.py", line 91, in _load_file
    self._tree = ast.parse(''.join(self._lines))
  File "C:\tools\miniconda3\envs\test\lib\ast.py", line 47, in parse
    return compile(source, filename, mode, flags,
  File "<unknown>", line 1
    def func_with_umlauts_äöü():
                               ^
SyntaxError: invalid character in identifier
```

**Comment by m-burst** [comment]
This is a problem in `flake8-plugin-utils` which we use internally.
I've opened an issue there (Afonasev/flake8-plugin-utils#36) and submitted a patch (Afonasev/flake8-plugin-utils#37).

**Comment by tobiasraabe** [comment]
Fantastic! Thank you very much!

**Comment by m-burst** [comment]
@tobiasraabe I've just released v1.2.3.
Could you please upgrade and check that the problem no longer occurs?

**Comment by tobiasraabe** [comment]
@m-burst It works like a charm :). Thank you very much for the quick responses and the fix!
